src/main/scala/mesosphere/marathon/api/v2/AppNormalization.scala

```diff
@@ -321,16 +333,29 @@ object AppNormalization {
     def mesosBridgeName: String
     def enabledFeatures: Set[String]
     def roleSettings: RoleSettings
+    def availableDeprecatedFeatures: DeprecatedFeatureSet
   }
 
   /** static app normalization configuration */
-  case class Configuration(defaultNetworkName: Option[String], override val mesosBridgeName: String, enabledFeatures: Set[String], roleSettings: RoleSettings) extends Config {
+  case class Configuration(
+      defaultNetworkName: Option[String],
+      override val mesosBridgeName: String,
+      enabledFeatures: Set[String],
+      roleSettings: RoleSettings,
+      availableDeprecatedFeatures: DeprecatedFeatureSet
```

Comment:
Rather than pass in this whole object, I might just pass in a Boolean indicating whether or not this feature is enabled